go.mod

```diff
@@ -20,7 +20,7 @@ require (
 	github.com/spf13/cobra v0.0.7
 	github.com/spf13/pflag v1.0.5
 	github.com/submariner-io/lighthouse v0.4.1
-	github.com/submariner-io/shipyard v0.4.1
+	github.com/submariner-io/shipyard v0.4.2-0.20200630133127-f7b9cae0e41d
```

Comment:
The final merged commit seems to be: 78b626b2e2a1e4254cfd568bef89673d7dac0a5 we need to update this ref before merging (I guess this is why it's in draft , so thanks for that hint).